### DISCUSSION TITLE
Is there a way to see what the models knows?

### DISCUSSION THREAD

**Discussion by mgatelabs** [nerdyrodent/VQGAN-CLIP#155] [Q&A]
Not sure if I'm using the right lingo, but is there a way to see the words/phrases the model knows?  So I can better write the words for it to describe the image.

**Comment by jonnm4** [comment]
Personally, I'd check out how google made / trained the ViT-B/16 or ViT-B/32 models that CLIP in this code is setup to use.
https://github.com/google-research/vision_transformer
